### PR TITLE
electri 10:02 - 11:40

### PR DESCRIPTION
**Pull request by H3arn**

### CODE REVIEW

File: electri/electri.ass [hunk 1]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
...
+Dialogue: 0,0:10:05.54,0:10:07.89,Chinese,,0,0,0,,并且认为  它的传输介质
 Dialogue: 0,0:10:07.89,0:10:11.40,English,,0,0,0,,either by fields or by the current in the wire.
-Dialogue: 0,0:10:07.89,0:10:11.40,Chinese,,0,0,0,,
+Dialogue: 0,0:10:07.89,0:10:11.40,Chinese,,0,0,0,,既可以是场  也可以是电线中的电流
 Dialogue: 0,0:10:11.40,0:10:13.58,English,,0,0,0,,But that is not the case,
-Dialogue: 0,0:10:11.40,0:10:13.58,Chinese,,0,0,0,,
+Dialogue: 0,0:10:11.40,0:10:13.58,Chinese,,0,0,0,,但事实并非如此
 Dialogue: 0,0:10:13.58,0:10:15.26,English,,0,0,0,,and people learned this the hard way
-Dialogue: 0,0:10:13.58,0:10:15.26,Chinese,,0,0,0,,
+Dialogue: 0,0:10:13.58,0:10:15.26,Chinese,,0,0,0,,人们是以艰难的方式明白这一点的
```
Comment (ksyx) [inline electri/electri.ass:522]:
```suggestion
Dialogue: 0,0:10:13.58,0:10:15.26,Chinese,,0,0,0,,人们认识到这一点花费的成本巨大
```

File: electri/electri.ass [hunk 2]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
...
 Dialogue: 0,0:10:15.26,0:10:18.61,English,,0,0,0,,when they started laying undersea telegraph cables.
-Dialogue: 0,0:10:15.26,0:10:18.61,Chinese,,0,0,0,,
+Dialogue: 0,0:10:15.26,0:10:18.61,Chinese,,0,0,0,,那是在他们开始铺设海底电报电缆的时候
 Dialogue: 0,0:10:18.61,0:10:22.71,English,,0,0,0,,The first Trans Atlantic cable was laid in 1858.
-Dialogue: 0,0:10:18.61,0:10:22.71,Chinese,,0,0,0,,
+Dialogue: 0,0:10:18.61,0:10:22.71,Chinese,,0,0,0,,第一条跨大西洋海底电缆在1858年铺设
```
Comment (ksyx) [inline electri/electri.ass:526]: Swap lines 524 526?

File: electri/electri.ass [hunk 3]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
...
 Dialogue: 0,0:10:22.71,0:10:24.25,English,,0,0,0,,- It only worked for about a month,
-Dialogue: 0,0:10:22.71,0:10:24.25,Chinese,,0,0,0,,
+Dialogue: 0,0:10:22.71,0:10:24.25,Chinese,,0,0,0,,它只运行了大约一个星期
```
Comment (ksyx) [inline electri/electri.ass:528]: *month*

File: electri/electri.ass [hunk 4]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
...
+Dialogue: 0,0:11:01.24,0:11:03.72,Chinese,,0,0,0,,携带这些能量和信息
 Dialogue: 0,0:11:03.72,0:11:05.01,English,,0,0,0,,And ultimately,
-Dialogue: 0,0:11:03.72,0:11:05.01,Chinese,,0,0,0,,
+Dialogue: 0,0:11:03.72,0:11:05.01,Chinese,,0,0,0,,而最终
 Dialogue: 0,0:11:05.01,0:11:07.44,English,,0,0,0,,this view proved correct.
-Dialogue: 0,0:11:05.01,0:11:07.44,Chinese,,0,0,0,,
+Dialogue: 0,0:11:05.01,0:11:07.44,Chinese,,0,0,0,,后者被证明正确
 Dialogue: 0,0:11:07.44,0:11:10.34,English,,0,0,0,,To insulate and protect the submarine cable,
-Dialogue: 0,0:11:07.44,0:11:10.34,Chinese,,0,0,0,,
+Dialogue: 0,0:11:07.44,0:11:10.34,Chinese,,0,0,0,,为了使电缆绝缘并被保护
```
Comment (ksyx) [inline electri/electri.ass:568]:
```suggestion
Dialogue: 0,0:11:07.44,0:11:10.34,Chinese,,0,0,0,,为了保护和绝缘电缆
```

File: electri/electri.ass [hunk 5]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
...
 Dialogue: 0,0:11:10.34,0:11:12.15,English,,0,0,0,,the central copper conductor
-Dialogue: 0,0:11:10.34,0:11:12.15,Chinese,,0,0,0,,
+Dialogue: 0,0:11:10.34,0:11:12.15,Chinese,,0,0,0,,电缆中部的铜导体
```
Comment (ksyx) [inline electri/electri.ass:570]:
```suggestion
Dialogue: 0,0:11:10.34,0:11:12.15,Chinese,,0,0,0,,电缆的铜芯
```

File: electri/electri.ass [hunk 6]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
...
+Dialogue: 0,0:11:23.90,0:11:27.38,Chinese,,0,0,0,,因为它增大了电缆的电容
 Dialogue: 0,0:11:27.38,0:11:31.09,English,,0,0,0,,This is why today, most power lines are suspended high up.
-Dialogue: 0,0:11:27.38,0:11:31.09,Chinese,,0,0,0,,
+Dialogue: 0,0:11:27.38,0:11:31.09,Chinese,,0,0,0,,这也是如今多数的电线都高高挂起的原因
 Dialogue: 0,0:11:31.09,0:11:33.88,English,,0,0,0,,Even the damp earth acts as a conductor,
-Dialogue: 0,0:11:31.09,0:11:33.88,Chinese,,0,0,0,,
+Dialogue: 0,0:11:31.09,0:11:33.88,Chinese,,0,0,0,,哪怕是潮湿的地面也会成为导体
 Dialogue: 0,0:11:33.88,0:11:36.76,English,,0,0,0,,so you want a large insulating gap of air
-Dialogue: 0,0:11:33.88,0:11:36.76,Chinese,,0,0,0,,
+Dialogue: 0,0:11:33.88,0:11:36.76,Chinese,,0,0,0,,所以  你会希望有一大层绝缘的空气挡在中间
```
Comment (ksyx) [inline electri/electri.ass:588]:
```suggestion
Dialogue: 0,0:11:33.88,0:11:36.76,Chinese,,0,0,0,,这样就可以有一大层绝缘的空气挡在中间
```

File: electri/electri.ass [hunk 7]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
 Dialogue: 0,0:09:58.04,0:10:01.42,English,,0,0,0,,that the energy flux is going in one direction.
 Dialogue: 0,0:09:58.04,0:10:01.42,Chinese,,0,0,0,,
 Dialogue: 0,0:10:02.26,0:10:05.54,English,,0,0,0,,You might think this is just an academic discussion
-Dialogue: 0,0:10:02.26,0:10:05.54,Chinese,,0,0,0,,
+Dialogue: 0,0:10:02.26,0:10:05.54,Chinese,,0,0,0,,你或许觉得  这只是一次学术讨论
```
Comment (ksyx) [inline electri/electri.ass:514]:
```suggestion
Dialogue: 0,0:10:02.26,0:10:05.54,Chinese,,0,0,0,,你或许觉得  这只是一场学术讨论
```

File: electri/electri.ass (hunk 2 above)
Comment (H3arn) [reply electri/electri.ass:526]: I suggest not?

File: electri/electri.ass (hunk 3 above)
Comment (H3arn) [reply electri/electri.ass:528]: sry I will fix this

Comment (H3arn) [reply electri/electri.ass:528]: could you edit this line? I can't edit this with your name appended

File: electri/electri.ass [hunk 8]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
...
+Dialogue: 0,0:10:32.65,0:10:34.84,Chinese,,0,0,0,,人们发现  在海底传送信号
 Dialogue: 0,0:10:34.84,0:10:37.22,English,,0,0,0,,over such a long distance under the sea,
-Dialogue: 0,0:10:34.84,0:10:37.22,Chinese,,0,0,0,,
+Dialogue: 0,0:10:34.84,0:10:37.22,Chinese,,0,0,0,,经过如此长的距离后
 Dialogue: 0,0:10:37.22,0:10:40.48,English,,0,0,0,,the pulses became distorted and lengthened.
-Dialogue: 0,0:10:37.22,0:10:40.48,Chinese,,0,0,0,,
+Dialogue: 0,0:10:37.22,0:10:40.48,Chinese,,0,0,0,,信号脉冲会失真并被拉长
 Dialogue: 0,0:10:40.48,0:10:43.57,English,,0,0,0,,It was hard to differentiate dots from dashes.
-Dialogue: 0,0:10:40.48,0:10:43.57,Chinese,,0,0,0,,
+Dialogue: 0,0:10:40.48,0:10:43.57,Chinese,,0,0,0,,难以辨别电码的点和横
```
Comment (635f0623-fd5e-5f7f-9ce9-6b476afb90c5) [inline electri/electri.ass:546]:
```suggestion
Dialogue: 0,0:10:40.48,0:10:43.57,Chinese,,0,0,0,,导致难以辨别电码的点和横
```

File: electri/electri.ass [hunk 9]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
...
+Dialogue: 0,0:11:23.90,0:11:27.38,Chinese,,0,0,0,,因为它增大了电缆的电容
 Dialogue: 0,0:11:27.38,0:11:31.09,English,,0,0,0,,This is why today, most power lines are suspended high up.
-Dialogue: 0,0:11:27.38,0:11:31.09,Chinese,,0,0,0,,
+Dialogue: 0,0:11:27.38,0:11:31.09,Chinese,,0,0,0,,这也是如今多数的电线都高高挂起的原因
 Dialogue: 0,0:11:31.09,0:11:33.88,English,,0,0,0,,Even the damp earth acts as a conductor,
-Dialogue: 0,0:11:31.09,0:11:33.88,Chinese,,0,0,0,,
+Dialogue: 0,0:11:31.09,0:11:33.88,Chinese,,0,0,0,,哪怕是潮湿的地面也会成为导体
 Dialogue: 0,0:11:33.88,0:11:36.76,English,,0,0,0,,so you want a large insulating gap of air
-Dialogue: 0,0:11:33.88,0:11:36.76,Chinese,,0,0,0,,
+Dialogue: 0,0:11:33.88,0:11:36.76,Chinese,,0,0,0,,这样就可以有一大层绝缘的空气挡在中间
```
Comment (635f0623-fd5e-5f7f-9ce9-6b476afb90c5) [inline electri/electri.ass:588]:
```suggestion
Dialogue: 0,0:11:33.88,0:11:36.76,Chinese,,0,0,0,,因此需要有一大层绝缘的空气挡在中间
```

File: electri/electri.ass [hunk 10]
```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
...
+Dialogue: 0,0:10:13.58,0:10:15.26,Chinese,,0,0,0,,人们是以艰难的方式明白这一点的
 Dialogue: 0,0:10:15.26,0:10:18.61,English,,0,0,0,,when they started laying undersea telegraph cables.
-Dialogue: 0,0:10:15.26,0:10:18.61,Chinese,,0,0,0,,
+Dialogue: 0,0:10:15.26,0:10:18.61,Chinese,,0,0,0,,那是在他们开始铺设海底电报电缆的时候
 Dialogue: 0,0:10:18.61,0:10:22.71,English,,0,0,0,,The first Trans Atlantic cable was laid in 1858.
-Dialogue: 0,0:10:18.61,0:10:22.71,Chinese,,0,0,0,,
+Dialogue: 0,0:10:18.61,0:10:22.71,Chinese,,0,0,0,,第一条跨大西洋海底电缆在1858年铺设
 Dialogue: 0,0:10:22.71,0:10:24.25,English,,0,0,0,,- It only worked for about a month,
-Dialogue: 0,0:10:22.71,0:10:24.25,Chinese,,0,0,0,,
+Dialogue: 0,0:10:22.71,0:10:24.25,Chinese,,0,0,0,,它只运行了大约一个月
```
Comment (635f0623-fd5e-5f7f-9ce9-6b476afb90c5) [inline electri/electri.ass:528]: “运行” &rarr; “投入使用”？

**Pull request merged**